docs/quickstart.md

```diff
@@ -27,7 +27,7 @@ $ mvn clean install -DskipTests -Dhive11
 {% include note.html content="Setup your local hadoop/hive test environment, so you can play with entire ecosystem. See [this](http://www.bytearray.io/2016/05/setting-up-hadoopyarnsparkhive-on-mac.html) for reference" %}
 
 
-## Supported Versions
+## Version Compatibility
```

Comment:
I think we should leave it as "Supported Versions" since its more standard parlance for this kind of stuff